client/my-sites/plugins/main.jsx

```diff
@@ -431,6 +432,7 @@ export default connect(
 
 		return {
 			selectedSite,
+			selectedSiteId: selectedSite && selectedSite.ID,
```

Comment:
Yeah, that was just a mixture of laziness and caution on my part (and maybe the realization that the code isn't getting that much nicer as long as we also drag along `selectedSite`). Since I'm still kinda wary of typos and the like happening in small-but-somewhat-unrelated changes like this, I'll defer to another PR 😁